src/app/app-extras.module.ts

```diff
@@ -1,21 +1,11 @@
-import {
-  NgModule
-} from '@angular/core';
+import { NgModule } from '@angular/core';
+import { SkyDocsToolsModule, SkyDocsToolsOptions } from '@skyux/docs-tools';
+import { SkyAppLinkModule } from '@skyux/router';
 
-import {
-  SkyDocsToolsModule,
-  SkyDocsToolsOptions
-} from '@skyux/docs-tools';
-
-import {
-  SkyAppLinkModule
-} from '@skyux/router';
+import { SkyActionHubModule } from './public/public_api';
 
 @NgModule({
-  exports: [
-    SkyAppLinkModule,
-    SkyDocsToolsModule
-  ],
+  exports: [SkyActionHubModule, SkyAppLinkModule, SkyDocsToolsModule],
   entryComponents: [],
```

Comment:
Very minor - and I'm not going to block this because of this. But we typically don't include entries here if they are empty. So `entryComponents` could be removed.